doc/api/events.md

```diff
@@ -169,6 +220,11 @@ const EventEmitter = require('events');
 All `EventEmitter`s emit the event `'newListener'` when new listeners are
 added and `'removeListener'` when existing listeners are removed.
 
+It supports the following option:
+
+* `autoCatch`, it can be `true` or `false`. It enables [automatic catch
```

Comment:
this property only able on `async` function, why don't rename it to `asyncCatch`? 🤔

Reply:
you could have a normal function that returns a promise

Reply:
@devsnek arguably a normal function that returns a promise is an async function :P